auth-connector/src/main/proto/usermanagement.proto

```diff
@@ -3893,9 +4057,181 @@ message UpdateUserRequest {
   // User.state will be set to ActorState.Value.ACTIVE if the value is false.
   // If it is not set then the user's state will not be changed.
   nullable.BoolValue isDeactivated = 7;
+  // The CRN of the identity provider connector the user belongs to.
+  // This field is deprecated and optional.
+  nullable.StringValue identityProviderCrn = 8 [(options.FieldExtension.hidden) = true,
+    (options.FieldExtension.hiddenReason) = "FOR_CLOUDERA_USE_ONLY",
+    (options.FieldExtension.deprecated) = true];
 }
 
 message UpdateUserResponse {
   // The updated user.
   User user = 1;
 }
+
+message SetCssoStrictModeRequest {
+  // The account ID for which strict mode is being enabled or disabled.
+  string accountId = 1;
+  // Whether to enable or disable strict mode on the account. The default value is 'false'.
+  // Set it 'true' to enable strict mode. Enabling strict mode will restrict Cloudera SSO login
+  // for all the users including account administrators of the account. Disabling strict mode
+  // will allow account administrator login through Cloudera SSO. The 'true' value also overrides
+  // the setting for `clouderaSSOLoginEnabled`.
+  bool enableStrictMode = 2;
+}
+
+message SetCssoStrictModeResponse {
+}
+
+message InteractiveLogoutRequest {
+  // The session token to be removed from server as a part of interactive logout process.
+  string sessionToken = 1 [(options.FieldExtension.sensitive) = true];
+  // The IP address that the request was made from.
+  string sourceIpAddress = 2;
+}
+
+message InteractiveLogoutResponse {
+}
+
+// ========================================================================
+// SECTION: SCIM
+//
+// These messages pertain to SCIM.
+// ========================================================================
+
+message ScimListUsersRequest {
```

Comment:
Do we also have to copy all defintions for scim?

Reply:
Yes, it's easier to keep the proto in sync by copying the whole file.